auth/providers/azure/graph/msi_tokenprovider_test.go

```diff
@@ -52,13 +53,25 @@ func TestMSITokenProvider(t *testing.T) {
 		if err != nil {
 			t.Fatalf("refresh should not return error: %s", err)
 		}
+		t.Logf("Response: %v", resp)
+
+		var expectedResp TokenResponse
+		err = json.Unmarshal([]byte(tokenResponse), &expectedResp)
+		if err != nil {
+			t.Fatalf("failed to unmarshal token response: %v", err)
+		}
 
 		if resp.Token != msiAccessToken {
 			t.Errorf("returned obo token '%s' doesn't match expected '%s'", resp.Token, msiAccessToken)
 		}
 		if resp.TokenType != expectedTokenType {
 			t.Errorf("expected token type: Bearer, actual: %s", resp.TokenType)
 		}
+
+		expectedExpiresOn, _ := strconv.Atoi(expectedResp.ExpiresOn)
+		if resp.ExpiresOn != expectedExpiresOn {
```

Comment:
To simplify this, can you have the expected `expires_on` value be a constant that gets used in the mock token response and here?

Reply:
Updated based on suggestion